charts/tekton-apps/Chart.yaml

```diff
@@ -13,7 +13,7 @@ triggersVersions: "v0.16.0"
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.2.0
+version: 0.2.1-dev.12
```

Comment:
stop using 0.2.x
use 0.1.x
latest version is 0.1.27, so use 0.1.28-dev.x

Reply:
Pls remove all 0.2.x tags and releases if they were built added to remote repo
https://github.com/saritasa-nest/saritasa-devops-helm-charts/tags

Reply:
That PR was based on @populov https://github.com/saritasa-nest/saritasa-devops-helm-charts/pull/51 where he has 0.2.0 version